>From 0213a6b552d954e4fc0beff85f92a54dc703cf66 Mon Sep 17 00:00:00 2001
From: Arnaud Patard <apatard@hupstream.com>
Date: Fri, 19 Oct 2012 20:05:38 +0200
Subject: [PATCH] Add Hyper-V modules to netboot and cdrom

Commit cd006086fa5d91414d8ff9ff2b78fbb593878e3c:

ata_piix: defer disks to the Hyper-V drivers by default

in mainline kernel and backported in debian kernel, it's not possible to use
ata_piix to access to hyper-v disk storage, which mean it's not possible
to install debian.
This commit is disabling ata_piix because the disk can be accessed with the
hyperv storage module. So, to install debian, the hyperv udeb is needed.

Signed-off-by: Arnaud Patard <apatard@hupstream.com>
---
 build/pkg-lists/cdrom/isolinux/amd64.cfg |    2 ++
 build/pkg-lists/cdrom/isolinux/i386.cfg  |    1 +
 build/pkg-lists/netboot/amd64.cfg        |    1 +
 build/pkg-lists/netboot/i386.cfg         |    1 +
 4 files changed, 5 insertions(+)

diff --git a/build/pkg-lists/cdrom/isolinux/amd64.cfg b/build/pkg-lists/cdrom/isolinux/amd64.cfg
index decee71..bc8dfca 100644
--- a/build/pkg-lists/cdrom/isolinux/amd64.cfg
+++ b/build/pkg-lists/cdrom/isolinux/amd64.cfg
@@ -9,3 +9,5 @@ sata-modules-${kernel:Version}
 pata-modules-${kernel:Version}
 firewire-core-modules-${kernel:Version}
 scsi-modules-${kernel:Version}
+hyperv-modules-${kernel:Version}
+
diff --git a/build/pkg-lists/cdrom/isolinux/i386.cfg b/build/pkg-lists/cdrom/isolinux/i386.cfg
index a164069..aba4ac3 100644
--- a/build/pkg-lists/cdrom/isolinux/i386.cfg
+++ b/build/pkg-lists/cdrom/isolinux/i386.cfg
@@ -10,3 +10,4 @@ pata-modules-${kernel:Version}
 firewire-core-modules-${kernel:Version}
 scsi-modules-${kernel:Version}
 usb-storage-modules-${kernel:Version}
+hyperv-modules-${kernel:Version}
diff --git a/build/pkg-lists/netboot/amd64.cfg b/build/pkg-lists/netboot/amd64.cfg
index ec6beef..ebbe064 100644
--- a/build/pkg-lists/netboot/amd64.cfg
+++ b/build/pkg-lists/netboot/amd64.cfg
@@ -7,6 +7,7 @@ nic-extra-modules-${kernel:Version}
 nic-usb-modules-${kernel:Version}
 nic-wireless-modules-${kernel:Version}
 virtio-modules-${kernel:Version}
+hyperv-modules-${kernel:Version}
 usb-modules-${kernel:Version}
 input-modules-${kernel:Version}
 console-setup-udeb
diff --git a/build/pkg-lists/netboot/i386.cfg b/build/pkg-lists/netboot/i386.cfg
index 3340b1d..680137a 100644
--- a/build/pkg-lists/netboot/i386.cfg
+++ b/build/pkg-lists/netboot/i386.cfg
@@ -7,6 +7,7 @@ nic-extra-modules-${kernel:Version}
 nic-usb-modules-${kernel:Version}
 nic-wireless-modules-${kernel:Version}
 virtio-modules-${kernel:Version}
+hyperv-modules-${kernel:Version}
 usb-modules-${kernel:Version}
 input-modules-${kernel:Version}
 console-setup-udeb
-- 
1.7.10.4

